docs/standard/microservices-architecture/architect-microservice-container-applications/containerize-monolithic-applications.md

```diff
@@ -34,7 +34,7 @@ From an infrastructure perspective, each server can run many applications within
 
 **Figure 4-2**. Monolithic approach: Host running multiple apps, each app running as a container
 
-Monolithic applications in Microsoft Azure can be deployed using dedicated VMs for each instance. Additionally, using [Azure VM Scale Sets](https://azure.microsoft.com/en-us/documentation/services/virtual-machine-scale-sets/), you can easily scale the VMs. [Azure App Service](https://azure.microsoft.com/en-us/services/app-service/) can also run monolithic applications and easily scale instances without requiring you to manage the VMs. Since 2016, Azure App Services can run single instances of Docker containers as well, simplifying deployment.
+Monolithic applications in Microsoft Azure can be deployed using dedicated VMs for each instance. Additionally, using [Azure VM Scale Sets](https://azure.microsoft.com/documentation/services/virtual-machine-scale-sets/), you can easily scale the VMs. [Azure App Service](https://azure.microsoft.com/services/app-service/) can also run monolithic applications and easily scale instances without requiring you to manage the VMs. Since 2016, Azure App Services can run single instances of Docker containers as well, simplifying deployment.
```

Comment:
change https://azure.microsoft.com/documentation/services/virtual-machine-scale-sets/ to https://docs.microsoft.com/azure/virtual-machine-scale-sets/